www/js/main.js

```diff
@@ -3512,7 +3512,14 @@ function showOptions( expandItem ) {
 					case "wtkey":
 						return true;
 					case "wto":
-						data = escapeJSON( $.extend( {}, unescapeJSON( data ), { key: page.find( "#wtkey" ).val() } ) );
+						data = unescapeJSON( data );
+						var newData = { key: page.find( "#wtkey" ).val() };
+
+						if ( newData.key && controller.settings.wto && controller.settings.wto.pws ) {
```

Comment:
I think you can revert this section. The overlayMap() callback will automatically add `pws` to `wto` if selected. So I dont think you need this conditional. The following code does this:

![image](https://user-images.githubusercontent.com/16738570/68548606-ac854d00-03e6-11ea-98e7-81122f7bb51f.png)

Reply:
Okay I think I understand and removed this now. Since WTO is populated correctly when options page loads and overlay map will correctly update the PWS and we now fixed the weather options which updates WTO, all cases should be covered.

Reply:
Yeah. Agree